lib/cc/engine/analyzers/command_line_runner.rb

```diff
@@ -0,0 +1,45 @@
+require "open3"
+require "timeout"
+
+module CC
+  module Engine
+    module Analyzers
+      class CommandLineRunner
+        DEFAULT_TIMEOUT = 20
+
+        def initialize(command, timeout = DEFAULT_TIMEOUT)
+          @command = command
+          @timeout = timeout
+        end
+
+        def run(input)
+          Timeout.timeout(timeout) do
+            Open3.popen3 command, "r+" do |stdin, stdout, stderr, wait_thr|
+              stdin.puts input
+              stdin.close
+
+              exit_code = wait_thr.value
+
+              output = stdout.gets
+              stdout.close
+
+              err_output = stderr.gets
+              stderr.close
+
+              if 0 == exit_code
```

Comment:
I noticed that there was a `exit_code.to_i` before the move. Is that still needed here?